channels/generic/http.py

```diff
@@ -1,31 +1,17 @@
 from channels.consumer import AsyncConsumer
 
+from ..exceptions import (
+    StopConsumer,
+)
```

Comment:
Keep it on one line, hopefully the lint stage will pass then and we'll be able to see the results of the test run on Travis CI :-)